cmd/arc/services/metamorph.go

```diff
@@ -65,6 +66,26 @@ func StartMetamorph(logger *slog.Logger) (func(), error) {
 		return nil, fmt.Errorf("invalid metamorph.processorCacheExpiryTime: %s", mapExpiryStr)
 	}
 
+	seenOnNetworkOlderThanStr, err := cfg.GetString("metamorph.checkSeenOnNetworkOlderThan")
```

Comment:
Instead of `GetString()` you can use `GetDuration()` then you don't need `time.ParseDuration()`